fil_actor_interface/src/builtin/power/mod.rs

```diff
@@ -94,6 +94,61 @@ impl State {
         }
     }
 
+    /// Returns the addresses of every miner that has claimed power in the power actor
```

Comment:
Could we avoid repeating the code by using a macro? To my understanding, the code differs only with types, no?

Reply:
done